payments/card-payments/after-payment.md

```diff
@@ -676,11 +676,11 @@ sequenceDiagram
 If you, for any reason, need to delete a paymentToken you use the
 `Delete payment token` request.
 
-> Please note that this call does not erase the card number stored at
-  Swedbank Pay. A card number is automatically deleted six months after a
-  successful `Delete payment token` request. If you want to remove card
-  information beforehand, you need to contact support.ecom@payex.com; and supply
-  them with the relevant transaction reference or payment token.
+> Please note that this call does not erase the card number stored at Swedbank
```

Comment:
Don't use `>` as it looks dull and is for quotes. Make this into a proper alert instead.